### ISSUE TITLE
Add NULL support to wkb

### ISSUE THREAD

**Issue by icco** [paulmach/orb#21]
The package `encoding/wkb` seems to have issues if a null point is passed into a scanner.

Example:
```go
package main

import (
	"database/sql"
	"log"

	_ "github.com/lib/pq"
	"github.com/paulmach/orb"
	"github.com/paulmach/orb/encoding/wkb"
)

func main() {
	db, _ := sql.Open("postgres", "postgres://localhost/writing?sslmode=disable")
	if err := db.Ping(); err != nil {
		log.Panic(err.Error())
	}

	row := db.QueryRow("SELECT ST_AsBinary(NULL::geometry)")
	var p orb.Point
	err := row.Scan(wkb.Scanner(&p))
	if err != nil {
		log.Panic(err.Error())
	}
}
```

Output:

```
$ go run main.go
2019/02/03 17:12:28 sql: Scan error on column index 0, name "st_asbinary": wkb: scan value must be []byte
panic: sql: Scan error on column index 0, name "st_asbinary": wkb: scan value must be []byte

goroutine 1 [running]:
log.Panic(0xc0000e9f58, 0x1, 0x1)
	/usr/local/Cellar/go/1.11.5/libexec/src/log/log.go:326 +0xc0
main.main()
	/Users/nat/tmp/example/main.go:22 +0x1f2
exit status 2
```

This is talking to a postgres 9.6 server.

**Comment by icco** [comment]
The use case for this is I have a data structure which has optional lat/long points attached to it.

**Comment by paulmach** [comment]
fix is here https://github.com/paulmach/orb/pull/22  let me know what you think.

**Comment by paulmach** [comment]
https://github.com/paulmach/orb/pull/22 has been merged and I tagged v0.1.2
